src/examples/crypto/blake2b/run.ts

```diff
@@ -1,16 +1,19 @@
 import { Bytes12, BLAKE2BProgram } from './blake2b.js';
+import { perfStart, perfEnd } from '../../../lib/testing/perf-regression.js';
 
-console.time('compile');
+const cs = await BLAKE2BProgram.analyzeMethods();
+perfStart('compile', BLAKE2BProgram.name, cs);
 await BLAKE2BProgram.compile();
-console.timeEnd('compile');
+perfEnd();
 
 let preimage = Bytes12.fromString('hello world!');
 
 console.log('blake2b rows:', (await BLAKE2BProgram.analyzeMethods()).blake2b.rows);
 
-console.time('prove');
+perfStart('prove', BLAKE2BProgram.name, cs, 'blake2b');
```

Comment:
Agree with what @Trivo25 suggested. Maybe let's enable a constructor for the performance checker that can be initialized with these parameters, then it can start and end at any desired time like: `checker.start(), checker.end()` or something.